### PR TITLE
Added Debian Jessie Installation instructions for ffmpeg

### PR DESCRIPTION
**Pull request by matt2005**
Adding instructions to use the avconv version of ffmpeg instead of compiling ffmpeg

### CODE REVIEW

File: source/_components/ffmpeg.markdown [hunk 1]
```diff
@@ -28,6 +28,18 @@ Configuration variables:
 - **ffmpeg_bin** (*Optional*): Set the ffmpeg binary (eg. `/usr/bin/ffmpeg`). Default 'ffmpeg'. 
 - **run_test** (*Optional*):  Check if `input` is usable by ffmpeg. Default True.
 
+### {% linkable_title Debian Jessie Installations %}
+To get the binary on Debian Jessie on a RPi you need to perform the following:
```
Comment (Landrash) [inline source/_components/ffmpeg.markdown]: Are you referring to Debain Jessie or Raspbian? Raspbian does some changes to Debian and depending on the base in question your instructions may fail.

Comment (matt2005) [inline source/_components/ffmpeg.markdown]: Raspbian Jessie lite

Comment (Landrash) [inline source/_components/ffmpeg.markdown]: I would suggest you edit the instructions to reflect this.

**Comment by matt2005** [comment]
Updated to include Raspbian Jessie lite

**Pull request merged**